enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -334,31 +400,56 @@ spec:
     port: 443
 ```
 
-In this configuration, the Gateway specifies that it uses the
-"istio-ingressgateway" service and defines three listeners with two different
-host names: `*.stage.example.com` and `*.prod.example.com`.  Suppose the
-"istio-ingressgateway" has the following service (again, some extraneous details
-are abbreviated):
+In this configuration, the Gateway defines three listeners with two different
+host names: `*.stage.example.com` and `*.prod.example.com`, as well as two
+different TCP ports: 80 and 443.  For this Gateway, Istio creates the following
+service (again, some extraneous details are abbreviated in the following
+example):
 
 ```yaml
 apiVersion: v1
 kind: Service
 metadata:
-  name: istio-ingressgateway
+  labels:
+    gateway.istio.io/managed: istio.io-gateway-controller
+  name: example-gateway
   namespace: openshift-ingress
+  ownerReferences:
+  - apiVersion: gateway.networking.k8s.io/v1alpha2
+    kind: Gateway
+    name: gateway
 spec:
   ports:
-  - name: http
+  - name: status-port
+    port: 15021
+    protocol: TCP
+  - name: stage-http
     port: 80
-  - name: https
+    protocol: TCP
+  - name: stage-https
     port: 443
+    protocol: TCP
+  selector:
+    istio.io/gateway-name: gateway
   type: LoadBalancer
 status:
   loadBalancer:
     ingress:
     - hostname: lb.example.com
 ```
 
+Note that the service defines each port once.  When multiple listeners specify
+the same port, Istio names the service port using the name of the first listener
+that specifies the port.  Multiplexing different host names using the same port
+number is done using TLS SNI or the HTTP `host` request header.  In addition,
+Istio adds a port named "status-port", which load balancers can use for health
+checks.  Istio automatically deletes the service when the Gateway is deleted.
```

Comment:
The automatic delete and the stuff in the Note below happen because of the automated deployment, right?

Reply:
I believe the specific mechanism is the garbage collector.  The service has an owner reference on the Gateway, so when the Gateway is deleted, the garbage collector should remove the service.

Reply:
I don't know if you want to add a reference to the open bug: https://issues.redhat.com/browse/OCPBUGS-8681

Reply:
I don't think the implementation details matter for this EP beyond that Istio does this step automatically.